partition/bootloader.go

```diff
@@ -55,6 +60,28 @@ type Bootloader interface {
 
 	// Name returns the bootloader name
 	Name() string
+
+	// configFile returns the name of the config file
+	ConfigFile() string
+}
+
+// InstallBootConfig install the bootloader config from the gadget
+// snap dir into the right place
+func InstallBootConfig(gadgetDir string) error {
+	for _, bl := range []Bootloader{&grub{}, &uboot{}} {
+		fn, err := find(gadgetDir, filepath.Base(bl.ConfigFile()))
```

Comment:
Oh.. why find?  Why would we accept that file to live anywhere inside the gadget instead of a very predictable and well named location?